packages/debian/entrypoint.sh

```diff
@@ -36,10 +36,27 @@ rpm2deb() {
 }
 
 
+buildrepo() {
+    set -x
+    mkdir -p /var/www/debian/{conf,incoming}
```

Comment:
Why use `/var/www/debian`? I'm confused, is it supposed to be served by some webserver?

Reply:
Also, if you want to set a basedir for `reprepro`, instead of duplicating the `-b` argument for each invocation, you could just set the `REPREPRO_BASE_DIR` environment variable.

Reply:
we donc need the `/var/www` for now, I will remove it